src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicatorTest.java

```diff
@@ -481,4 +494,243 @@ public void testDoneReadingWithExceptionAndHasCompletionPacketLeftToRead() throw
+	 * Test AxisCommunicator#getMultipleStatistics success
+	 * Expect device info get data success
+	 */
+	@Test
+	public void testAverPTZCommunicatorDeviceInfo() {
+		extendedStatistic = (ExtendedStatistics) averPTZCommunicator.getMultipleStatistics().get(0);
+		stats = extendedStatistic.getStatistics();
+
+		Assert.assertEquals("AVer Information Co.", stats.get(StatisticsProperty.DEVICE_INFORMATION.getName() + HASH + StatisticsProperty.DEVICE_MFG.getName()));
+		Assert.assertEquals("PTZ330", stats.get(StatisticsProperty.DEVICE_INFORMATION.getName() + HASH + StatisticsProperty.DEVICE_MODEL.getName()));
+		Assert.assertEquals("5310505800460", stats.get(StatisticsProperty.DEVICE_INFORMATION.getName() + HASH + StatisticsProperty.DEVICE_SERIAL_NUMBER.getName()));
+		Assert.assertEquals("0.0.0003.72", stats.get(StatisticsProperty.DEVICE_INFORMATION.getName() + HASH + StatisticsProperty.DEVICE_FIRMWARE_VERSION.getName()));
+		Assert.assertEquals("6", stats.get(StatisticsProperty.DEVICE_INFORMATION.getName() + HASH + StatisticsProperty.DEVICE_LAST_PRESET_RECALLED.getName()));
+	}
+
+	/**
+	 * Test AxisCommunicator#getMultipleStatistics success
+	 * Expect power status get data success
+	 */
+	@Test
+	public void testAverPTZCommunicatorPowerStatus() {
+		extendedStatistic = (ExtendedStatistics) averPTZCommunicator.getMultipleStatistics().get(0);
+		advancedControllableProperties = extendedStatistic.getControllableProperties();
+
+		for (AdvancedControllableProperty property : advancedControllableProperties) {
+			if (property.getName().equalsIgnoreCase(Command.POWER.getName())) {
+				Assert.assertTrue((int) property.getValue() == 1 || (int) property.getValue() == 0);
+			}
+		}
+	}
+
+	/**
+	 * Test AxisCommunicator#getMultipleStatistics success
+	 * Expect focus mode get data success
+	 */
+	@Test
+	public void testAverPTZCommunicatorFocusMode() {
+		extendedStatistic = (ExtendedStatistics) averPTZCommunicator.getMultipleStatistics().get(0);
+		advancedControllableProperties = extendedStatistic.getControllableProperties();
+
+		for (AdvancedControllableProperty property : advancedControllableProperties) {
+			if (property.getName().equalsIgnoreCase(Command.FOCUS.getName() + HASH + Command.FOCUS_MODE.getName())) {
+				Assert.assertTrue((int) property.getValue() == 1 || (int) property.getValue() == 0);
+			}
+		}
+	}
+
+	/**
+	 * Test AxisCommunicator#getMultipleStatistics success
+	 * Expect backlight status get data success
+	 */
+	@Test
+	public void testAverPTZCommunicatorBacklightStatus() {
+		extendedStatistic = (ExtendedStatistics) averPTZCommunicator.getMultipleStatistics().get(0);
+		advancedControllableProperties = extendedStatistic.getControllableProperties();
+
+		for (AdvancedControllableProperty property : advancedControllableProperties) {
+			if (property.getName().equalsIgnoreCase(Command.EXPOSURE.getName() + HASH + Command.BACKLIGHT.getName())) {
+				Assert.assertTrue((int) property.getValue() == 1 || (int) property.getValue() == 0);
```

Comment:
this value is dynamic or static? Why can't we have a specific value with its current value?

Reply:
this control is switch control type and it has a static value 0, 1. So I think we can do it.